public/src/components/calls/active_call_control.jsx

```diff
@@ -31,37 +39,50 @@ export default class ActiveCallControl extends Component {
     }
     updateCallOutcome(text.toUpperCase());
   }
-
+  syncOutcomeInForm(e) {
+    const { change, form } = this.props;
+    change(form, 'outcome', e.currentTarget.value);
+  }
   render() {
-    const { current_call_contact_name, handleSubmit, outcome, status } = this.props;
+    const { current_call_contact_name,
+            handleSubmit,
+            outcome,
+            status,
+            form_errors } = this.props;
     const outcomes = [
       {
-        value: 'Answered',
+        value: 'ANSWERED',
+        label: 'Answered',
```

Comment:
I added labels to the toolbar so that the value would match the enum in the db. Also, I made a separate prop, `label` so that the button label independent from value.